core/src/main/res/values/strings.xml

```diff
@@ -765,6 +765,14 @@
     <string name="rating_later_label">Remind me later</string>
     <string name="rating_now_label">Sure, let\'s do this!</string>
 
+    <!-- Share episode dialog -->
+    <string name="share_episode_label">Share…</string>
+    <string name="share_episode_positive_label_button">Share</string>
+    <string name="share_dialog_include_label">Include:</string>
+    <string name="share_start_at_timer_dialog_label">Start at (timer)</string>
```

Comment:
What about `Playback position`?

Reply:
are you refering to rename "share_start_at_timer_dialog_label"  for "share_playback_position_dialog_label"? 

Sorry , I don't catch this comment. Can you explain a little more please :)

Reply:
Sorry, my comments are getting shorter and shorter in the past few weeks. That has lead to some issues with another PR already. I'm pretty busy and try to squeeze in some AntennaPod work.

I meant the text of the string that is shown to the user. I think that "Playback position" might be easier to understand than "Start at (timer)". Not completely sure, though. I think the word "timer" is only used for the sleep timer currently. How does YouTube name that? I think they also use something like "Include position" or "Include timestamp"

Reply:
Thanks, don't worry. 

YouTube just called it "Start at" (without 'timer' word) if you prefer I can delete the word 'timer' or just use "Playback position". I like the last one , I agree, it is pretty easy to understand

Reply:
Good point mentioning YouTube. I think we are in a bit different situation here, though. Just including the position with AntennaPod does not automatically mean that the episode is actually played from that position. I have a feeling that it is probably not done in most cases. So "Playback position" could be the better choice, here, I think.

Reply:
You mentioned YouTube first, @ByteHamster ;)
I'd put my vote to 'Playback position'

Reply:
> You mentioned YouTube first, @ByteHamster ;)

Oops :smile: I currently have a few other things in mind and somehow completely forgot that...

Reply:
no problem. I like "Playback position". I have added it :)